src/ot_orchestration/dags/genetics_dag.py

```diff
@@ -0,0 +1,26 @@
+"""Airflow DAG for GWAS Catalog processing."""
+
+from __future__ import annotations
+
+from ot_orchestration.utils import read_yaml_config
+from ot_orchestration.utils.common import shared_dag_args, shared_dag_kwargs
+from airflow.models.dag import DAG
+from pathlib import Path
+
+import logging
+from ot_orchestration.dynamic_dags import generic_genetics_dag
+
+
+config_path = "/opt/airflow/config/config.yaml"
```

Comment:
well, I wanted it to be external, but in the current setup it can as well sit in the package as the PIS config. I will definately change the name though